prometheus_pgbouncer_exporter/collector.py

```diff
@@ -150,6 +161,23 @@ def _exportMetrics(self, results, metricPrefix, metricMappings, labelMappings, e
 
         return metrics
 
+    def _exportKeyValueMetrics(self, results, metricPrefix, metricMappings):
+        metrics = []
+
+        for result in results:
+            for mapping in metricMappings:                
+                if (result["key"] == mapping["key"]):
+                    value = result["value"]   
```

Comment:
This assignment is not needed, you can use `result["value"]` directly in the code below

```suggestion
```

Reply:
It should be ok with new commit